srcpkgs/avizo/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'avizo'
+pkgname=avizo
+version=1.2.1
+revision=1
+build_style=meson
+hostmakedepends="vala pkg-config cmake gobject-introspection glib-devel gdk-pixbuf"
+makedepends="gtk+3-devel gtk-layer-shell-devel glib-devel"
+depends="bash light pamixer"
+short_desc="Neat notification daemon"
+maintainer="Bnyro <bnyro@tutanota.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/misterdanb/avizo"
+distfiles="https://github.com/misterdanb/avizo/archive/refs/tags/${version}.tar.gz"
+checksum=efd4763db7c802d22e8939dac909d4886f5967aefbfc3fd526735da2a814b91b
+
+post_patch() {
+	sed -i 's@#!/bin/sh@#!/bin/bash@g'  lightctl volumectl
```

Comment:
why? in `lightctl` i see "# This script is POSIX Shell Command Language compliant."   and it looks correct,

Reply:
Good catch. No clue why I added that, I tested it and can confirm it works with `/bin/sh` as well.